### PR TITLE
Sign get plz script

### PR DESCRIPTION
**Pull request by peterebden**
Makes it possible to GPG verify it before running.

### CODE REVIEW

File: docs/faq.html [hunk 1]
```diff
@@ -51,6 +51,39 @@ <h2 class="title-2">
   </p>
 </section>
 
+<section class="mt4">
+  <h2 class="title-2"><a name="verify">How can I verify the download of Please?</a></h2>
```
Comment (tiagovtristao) [inline docs/faq.html:55]: nit: `<h2 id="verify" class="title-2">How can I verify the download of Please?</h2>` will make it consistent with similar usages in other places.

File: docs/quickstart.html [hunk 2]
```diff
@@ -58,8 +58,12 @@ <h2 class="title-2">Installing</h2>
     <code class="code">~/.please</code> but you can put it where you want.
   </p>
 
+  <p>See <a href="faq.html#verify">here</a> for information if you'd like to verify
+    signatures of the download before running.</a>
```
Comment (tiagovtristao) [inline docs/quickstart.html:62]: `</a>` => `</p>`

**Review by tiagovtristao — APPROVED** [review]
Looks good, just a minor tweak. Thanks.

File: docs/faq.html (hunk 1 above)
Comment (peterebden) [reply docs/faq.html]: Ah thanks, I never knew that worked that way!

File: docs/quickstart.html (hunk 2 above)
Comment (peterebden) [reply docs/quickstart.html]: I do know the difference between `a` and `p`, honestly... :-P

**Pull request merged**